### ISSUE TITLE
Alt text for vignettes

### ISSUE THREAD

**Issue by fweber144** [stan-dev/projpred#526]
When building the pkgdown site, the following notes are shown:
```
Reading vignettes/latent.Rmd
Writing articles/latent.html
# [...]
✖ Missing alt-text in vignettes/latent.Rmd
• latent_files/figure-html/plot_vsel_lat-1.png
• latent_files/figure-html/post_vs_trad-1.png
• latent_files/figure-html/plot_vsel_nebin-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
Reading vignettes/projpred.Rmd
Writing articles/projpred.html
✖ Missing alt-text in vignettes/projpred.Rmd
• projpred_files/figure-html/plot_vsel_fast-1.png
• projpred_files/figure-html/plot_vsel_fast_refit-1.png
• projpred_files/figure-html/plot_vsel-1.png
• projpred_files/figure-html/plot_cv_proportions-1.png
• projpred_files/figure-html/plot_cv_proportions_cumul-1.png
• projpred_files/figure-html/bayesplot_prj-1.png
• projpred_files/figure-html/bayesplot_ref-1.png
• projpred_files/figure-html/proj_predict-1.png
ℹ Learn more in `vignette(pkgdown::accessibility)`.
```
I'm currently lacking the time to add alt text, but we should do this in the future.
